tck/api/src/main/java/io/astefanutti/metrics/cdi/se/ConcurrentGaugedMethodBeanTest.java

```diff
@@ -64,33 +64,33 @@
     private MetricRegistry registry;
 
     @Inject
-    private CountedMethodBean<Long> bean;
+    private ConcurrentGaugedMethodBean<Long> bean;
 
     @Test
     @InSequence(1)
     public void countedMethodNotCalledYet() {
-        assertThat("Counter is not registered correctly", registry.getCounters(), hasKey(COUNTER_NAME));
-        Counter counter = registry.getCounters().get(COUNTER_NAME);
+        assertThat("Concurrent Gauges is not registered correctly", registry.getConcurrentGauges(), hasKey(C_GAUGE_NAME));
+        ConcurrentGauge counter = registry.getConcurrentGauges().get(C_GAUGE_NAME);
 
         // Make sure that the counter hasn't been called yet
-        assertThat("Counter count is incorrect", counter.getCount(), is(equalTo(COUNTER_COUNT.get())));
+        assertThat("Concurrent Gauges count is incorrect", counter.getCount(), is(equalTo(COUNTER_COUNT.get())));
     }
 
     @Test
     @InSequence(2)
-    public void countedMethodNotCalledYet(@Metric(name = "countedMethod", absolute = true) Counter instance) {
-        assertThat("Counter is not registered correctly", registry.getCounters(), hasKey(COUNTER_NAME));
-        Counter counter = registry.getCounters().get(COUNTER_NAME);
+    public void countedMethodNotCalledYet(@Metric(name = "countedMethod", absolute = true) ConcurrentGauge instance) {
```

Comment:
The metric name here should be `C_GAUGE_NAME`?

Reply:
right, it should be changed to `C_GAUGE_NAME`